src/components/DocumentRenderer/LocalDocumentRendererContainer.tsx

```diff
@@ -27,7 +29,12 @@ export const LocalDocumentRendererContainer: FunctionComponent<NavigationProps>
         document={document.document}
         goBack={() => navigation.goBack()}
       />
-      <DocumentDetailsSheet document={document.document} />
+      <DocumentDetailsSheet
+        document={document.document}
+        onGenerateQr={generateQr(document.document)}
+        qrCode={qrCode}
+        qrCodeLoading={qrCodeLoading}
```

Comment:
Hmm, given that all the qr-related props are being passed down, should we let `DocumentDetailsSheet` handle the qr related stuff instead?